variantlib/variants_json.py

```diff
@@ -79,7 +80,16 @@ def to_str(self) -> str:
                 vhash: vdesc.to_dict() for vhash, vdesc in self.variants.items()
             },
         }
-        return json.dumps(data, indent=4)
+
+        if not data[VARIANT_INFO_DEFAULT_PRIO_KEY][VARIANT_INFO_FEATURE_KEY]:
+            # Remove empty feature priorities
+            del data[VARIANT_INFO_DEFAULT_PRIO_KEY][VARIANT_INFO_FEATURE_KEY]
+
+        if not data[VARIANT_INFO_DEFAULT_PRIO_KEY][VARIANT_INFO_PROPERTY_KEY]:
+            # Remove empty property priorities
+            del data[VARIANT_INFO_DEFAULT_PRIO_KEY][VARIANT_INFO_PROPERTY_KEY]
+
+        return json.dumps(data, indent=4, sort_keys=True, ensure_ascii=True)
```

Comment:
```suggestion
        return json.dumps(data, indent=4, sort_keys=True)
```

Forcing ASCII is incorrect here (and probably some of our regexes are incorrect too). For example, Python classes can use Unicode names.

Reply:
Our regex AFAIK strictly allows  ASCII chars. And if it doesn't we should fix that. 
I don't want Non ASCII chars in any of the variant stuff.

A variant `👍` makes no sense